fastlane_bot/bot.py

```diff
@@ -357,9 +357,24 @@ def _convert_trade_instructions(
             for ti in errorless_trade_instructions_dic
             if ti is not None
         )
+        result = self._add_strategy_id_to_trade_instructions_dic(result)
         result = [TradeInstruction(**ti) for ti in result]
         return result
 
+    def _add_strategy_id_to_trade_instructions_dic(
+        self, trade_instructions_dic: Generator
+    ) -> list:
+        lst = []
+        for ti in trade_instructions_dic:
+            cid = ti["cid"]
+            if '-' in cid:
+                cid = cid.split('-')[0]
+            ti["strategy_id"] = self.db.get_pool(
+                cid=cid
```

Comment:
Can skip the 'if '-' in cid' above, and just do `cid=cid.split('-')[0]` here.

Reply:
this has been corrected